style/ruby/README.md

```diff
@@ -31,6 +31,15 @@ In additional to the following guidelines, please also use the [.rubocop.yml](./
     ```
 - Prefer `size` over `count` or `length`. [Reference](https://stackoverflow.com/a/6083229/2456549).
 
+- Use `ENV.fetch('VARNAME')` instead of ENV. This will raise an error on deployment so developers can easily tell when they are missing ENVs.
+```ruby
```

Comment:
This needs to be indented with the rest of the code. Also, add a space between the list item and code block.